accountsplus/admin.py

```diff
@@ -185,7 +183,7 @@ def reset_passwords(self, request, queryset):
     reset_passwords.short_description = 'Send password reset emails to selected Users'
 
     def get_timezone(self, obj):
-        return unicode(obj.timezone)
+        return str(obj.timezone)
 
     def masquerade(self, obj):
         return '<a href="{}">sign in</a>'.format(django.urls.reverse('masquerade', kwargs={'user_id': obj.id}))
```

Comment:
@waseem-omar  I think we need to mark this as `SafeString` so that it appears as hyperlink in the table

Reply:
Should look like this `django.utils.html.mark_safe('<a href="{}">sign in</a>'.format(django.urls.reverse('masquerade', kwargs={'user_id': obj.id})))`

Reply:
Done @nasief, I have also refactored Masquerading views as you suggested above.